website/content/en/status/report-2023-07-2023-09/mfsbsd.adoc

```diff
@@ -12,19 +12,22 @@ Contact: Soobin Rho <soobinrho@FreeBSD.org>
 "mfsBSD is a toolset to create small-sized but full-featured mfsroot based distributions of FreeBSD that store all files in memory (MFS) [Memory File System] and load from hard drive, USB storage device, or optical medium.
 It can be used for a variety of purposes, including diskless systems, recovery partitions and remotely overwriting other operating systems."
 
-mailto:mm@FreeBSD.org[Martin Matuska] is the creator of mfsBSD, and he is also the author of the original link:https://people.freebsd.org/~mm/mfsbsd/mfsbsd.pdf[mfsBSD white paper], which is where the excerpt above was taken from.
-The upstream mfsBSD is maintained at the link:https://github.com/mmatuska/mfsbsd[GitHub mfsBSD repository].
+http://blog.vx.sk/[Martin Matuška] is the creator of mfsBSD.
```

Comment:
We usually link to e-mail addresses on names in status reports. Moreover, the linked blog has many references to different things that are not about mfsBSD, nor even about FreeBSD.

Please put the e-mail address back.

Reply:
Thanks, resolved by [`aa9fcb7` (#257)](https://github.com/freebsd/freebsd-doc/pull/257/commits/aa9fcb7e81e9f0fc970bb6b536eba6017712a63c). 

Would you like the address for `mm@` to be visible? (As it is for `soobinrho@` at line 8.)

Reply:
No, thanks, no need to make it visible.

Reply:
All clear. I'll probably remember all of this for future mfsBSD reports.